<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Create_Dash_Account</class>
 <widget class="QDialog" name="Create_Dash_Account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color: #0000ff;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ff0000;&quot;&gt;Welcome to Auto Savior !!!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#0000ff;&quot;&gt;Please respect the fallowing rules to create your Savior account:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#000000;&quot;&gt;*&lt;/span&gt;&lt;span style=&quot; font-size:9pt; color:#0000ff;&quot;&gt;The Master Password can be as long or as short as you want.&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#000000;&quot;&gt;*&lt;/span&gt;&lt;span style=&quot; font-size:9pt; color:#0000ff;&quot;&gt;The use of following typographical characters is allowed :&lt;br/&gt;~ ! @ # $ % &amp;amp; * ( ) _ + - = { } | [ ] \ : ; &amp;lt; &amp;gt; ? , . /.&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#000000;&quot;&gt;*&lt;/span&gt;&lt;span style=&quot; font-size:9pt; color:#0000ff;&quot;&gt;The email &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ff0000;&quot;&gt;MUST BE UNIQUE TO EACH ACCOUNT&lt;/span&gt;&lt;span style=&quot; font-size:9pt; color:#0000ff;&quot;&gt; created.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="firstName">
           <property name="styleSheet">
            <string notr="true">color: #0000ff;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>First Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lastName">
           <property name="styleSheet">
            <string notr="true">color: #0000ff;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Last Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Username">
           <property name="styleSheet">
            <string notr="true">color: #0000ff;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="pwd">
           <property name="styleSheet">
            <string notr="true">color: #0000ff;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="c_pwd">
           <property name="styleSheet">
            <string notr="true">color: #0000ff;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Confirm Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="email">
           <property name="styleSheet">
            <string notr="true">color: #0000ff;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="line_firstName"/>
         </item>
         <item>
          <widget class="QLineEdit" name="line_lastName"/>
         </item>
         <item>
          <widget class="QLineEdit" name="line_Username"/>
         </item>
         <item>
          <widget class="QLineEdit" name="line_pwd">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_c_pwd">
           <property name="inputMask">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_email">
           <property name="inputMask">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_pic">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/pic/registrationICON.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Register">
       <property name="styleSheet">
        <string notr="true">color: #0000ff;
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="styleSheet">
        <string notr="true">color: #0000ff;
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
